patch-package
--- a/node_modules/react-native-animatable/registry.js
+++ b/node_modules/react-native-animatable/registry.js
@@ -16,9 +16,11 @@ export function getAnimationNames() {
 
 export function initializeRegistryWithDefinitions(definitions) {
   Object.keys(definitions).forEach(animationName => {
-    registerAnimation(
-      animationName,
-      createAnimation(definitions[animationName]),
-    );
+    if (animationName !== '__esModule') {
+      registerAnimation(
+        animationName,
+        createAnimation(definitions[animationName]),
+      );
+    }
   });
 }
